src/libs/OnyxSelectors/reportWithoutHasDraftSelector.ts

```diff
@@ -1,9 +1,14 @@
-import type {OnyxValue} from '@src/ONYXKEYS';
+import type {OnyxEntry} from 'react-native-onyx';
+import type {Report} from '@src/types/onyx';
 
-export default function reportWithoutHasDraftSelector(report: OnyxValue<'report_'>) {
+type ReportWithoutHasDraft = Omit<Report, 'hasDraft'>;
+
+export default function reportWithoutHasDraftSelector(report: OnyxEntry<Report>): OnyxEntry<ReportWithoutHasDraft> {
     if (!report) {
         return report;
     }
```

Comment:
```suggestion
    if (!report) {
        return null;
    }
```

NAB: this condition looks crazy, could we make it more clear?